l5kit/l5kit/dataset/agent.py

```diff
@@ -36,34 +40,27 @@ def load_agents_mask(self) -> np.ndarray:
         Returns: a boolean mask of the same length of the dataset agents
 
         """
-        history_num_frames = self.cfg["model_params"]["history_num_frames"]
-        future_num_frames = self.cfg["model_params"]["future_num_frames"]
         agent_prob = self.cfg["raster_params"]["filter_agents_threshold"]
 
-        group_name = f"{history_num_frames}_{future_num_frames}_{agent_prob}"
         try:
-            agents_mask = self.dataset.root[f"agents_mask/{group_name}"]
+            agents_mask = self.dataset.root[f"agents_mask/{agent_prob}"]
         except KeyError:
             print(
-                f"cannot find {group_name} in {self.dataset.path},\n"
-                f"your cfg has loaded history_num_frames={history_num_frames}, future_num_frames={future_num_frames} "
-                f"and filter_agents_threshold={agent_prob};\n"
-                "but those values don't have a match among the agents_mask in the zarr\n"
-                "Mask will now be generated for these parameters."
+                f"cannot find the right config in {self.dataset.path},\n"
```

Comment:
Minor: do we really want to throw an error for this or just a warning?

Reply:
it's not throwing an error actually , just printing a warning before computing the masks

Reply:
I think we should swich to logger to have proper warning at some point